README.md

```diff
@@ -1,4 +1,5 @@
-# Mapotempo Optimizer API [![Build Status](https://travis-ci.com/Mapotempo/optimizer-api.svg?branch=master)](https://travis-ci.com/Mapotempo/optimizer-api)
+# Mapotempo Optimizer API ![dev Build Status](https://github.com/Mapotempo/optimizer-api/actions/workflows/main.yml/badge.svg?branch=dev)
```

Comment:
why do you replace master branch by dev branch?

Reply:
I thought it was interesting to specify the dev, but I will move it back to master branch.